components/footer.js

```diff
@@ -59,6 +65,7 @@ const Footer = ({ intl }) => (
           <Flex flexWrap='wrap' alignItems='center'>
             <Box p={[1, 0]} mb={[0, 3]} width={[1/2, 1]}>
               <ExplorerLogo height='32px' />
+              <small><FooterVersion>{intl.formatMessage({ id: 'Footer.Text.Version' }) }: {version}</FooterVersion></small>
```

Comment:
It might be a bit confusing to put the version number right under 'OONI' which represents the probe or the entire ecosystem of tools, and the version number itself is of the 'Explorer'. We can show the version number in a further less prominent spot. You can choose from:
* right under the Creative Commons license notice
* on the far right end of the copyright and license notice, as a sibling node to [this `<FooterBox>`](https://github.com/ooni/explorer/blob/master/components/footer.js#L89). You can add `justifyContent='space-between'` to the containing `<Flex>`

Reply:
Thanks for pointing this out. Makes sense. Going to fix it now.

Reply:
I think it looks better under the Creative Commons rather than far end since it kind of clashes with the feedback icon.

Following are the screenshots:

Under:
![Screenshot-2019-10-2 OONI Explorer - Open Data on Internet Censorship Worldwide(1)](https://user-images.githubusercontent.com/9530293/66062375-f153d380-e55e-11e9-95b1-59144f4d491b.png)

Far right end:
![Screenshot-2019-10-2 OONI Explorer - Open Data on Internet Censorship Worldwide](https://user-images.githubusercontent.com/9530293/66062387-f87ae180-e55e-11e9-8924-d16d6fe0cf21.png)

Reply:
Also, I was wondering if `OONI Explorer Version: ` would be a a better text than just `Version` to make it clear.

Reply:
Please go ahead with putting it under the license notice.

> wondering if OONI Explorer Version: would be a a better text 
You can leave it as 'Version' for now. The rendered text will be generated from a spreadsheet which we coordinate with our translators. The English version will also be edited by someone. Thanks for considering.